src/Compilers/CSharp/Portable/Symbols/NamedTypeSymbol.cs

```diff
@@ -1326,10 +1326,11 @@ public override bool IsTupleCompatible(out int tupleCardinality)
 
                     do
                     {
+                        Debug.Assert(!typeToCheck.IsDefinition);
```

Comment:
Yes, it is.

---
In reply to: [76453513](https://github.com/dotnet/roslyn/pull/13392#discussion_r76453513) [](ancestors = 76453513)